.eslintrc.json

```diff
@@ -1,197 +1,92 @@
 {
-  "env": {
-    "es2021": true,
-    "node": true,
-    "jest": true
-  },
-  "extends": [
-    "standard"
-  ],
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "ecmaVersion": 12,
-    "sourceType": "module"
-  },
-  "plugins": [
-    "@typescript-eslint"
-  ],
-  "rules": {
-    "indent": 0,
-    "@typescript-eslint/indent": [
-      "error",
-      2
-    ]
-  },
   "overrides": [
     {
-      "files": [
-        "*.graphql"
-      ],
+      "files": ["*.js", "*.ts"],
+      "env": {
+        "es2021": true,
+        "node": true,
+        "jest": true
+      },
+      "extends": ["standard"],
+      "parser": "@typescript-eslint/parser",
```

Comment:
It's very important that you don't have any rules configured at the top-level config (like `env`, `extends`, `parser`, `rules`, `parserOptions`), and move all configurations into the overrides section. Since JavaScript rules can't run on GraphQL files and vice versa, if you have rules configured at the top-level, they will try to also execute for all overrides, as ESLint's configs cascade.